packages/rest/src/rest.server.ts

```diff
@@ -229,6 +238,16 @@ export class RestServer extends Context implements Server, HttpServerLike {
     this.bind(RestBindings.HANDLER).toDynamicValue(() => this.httpHandler);
   }
 
+  protected _setupOASEnhancerIfNeeded() {
+    if (this._OASEnhancer != null) return;
```

Comment:
Nitpick: we could use `this._OASEnhancer = this.getSync(OAS_ENHANCER_SERVICE, {optional: true});` to check so that the service can be bound to the server.